src/editors/playgroundController.ts

```diff
@@ -573,7 +573,11 @@ export default class PlaygroundController {
 
     if (evaluateResponse?.outputLines?.length) {
       for (const line of evaluateResponse.outputLines) {
-        this._outputChannel.appendLine(line.content);
+        this._outputChannel.appendLine(
+          typeof line.content === 'string'
+            ? line.content
+            : JSON.stringify(line.content)
```

Comment:
Is `JSON.stringify()` the right stringifier here? We’re showing this in the output to humans, maybe something like `util.inspect()` would make more sense, if a user really wants JSON output they can always use `print(EJSON.stringify(...))` (or [hopefully soon](https://jira.mongodb.org/browse/MONGOSH-1361) `printejson(...)`)?

Reply:
I like the idea of using util.inspect() here 👍

Reply:
Yes it definitely would, thanks for the suggestion.
Updated and added a test.